stdlib/public/core/ContiguousCollection.swift

```diff
@@ -0,0 +1,34 @@
+//===--- ContigousCollection.swift ----------------------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+
```

Comment:
@airspeedswift this is what we are expecting for `ContiguousCollection`. Let me know if we need to adjust this.